src/contrib/cluster/Akka.Cluster.Metrics.Tests.MultiNode/ClusterMetricsExtensionSpec.cs

```diff
@@ -93,7 +91,7 @@ private async Task Should_collect_and_publish_metrics_and_gossip_them_around_the
         {
             await AwaitAssertAsync(() =>
             {
-                AwaitClusterUp(roles: _config.NodeList.ToArray());
+                AwaitClusterUp(Roles.ToArray());
```

Comment:
@Aaronontheweb Could you give me a hint for the future: what is the difference here? Isn't `Roles` just same list of roles stored internally?
I see that I am using this `Roles` in another spec that passes without issues, and maybe this will work here too - but why?

Reply:
Roles is something that gets populated automatically by the `MultiNodeSpec` base class - the content of the list is ordered and populated by the base classes themselves. The issue was that the `HashSet<T>` you were using could change the roles slightly since I think the `HashCode`s were different on a process by process basis.

Reply:
You are right, or course, set does not guarantee same ordering... Great catch, @Aaronontheweb , thanks a lot!